peps/pep-9999.rst

```diff
@@ -0,0 +1,120 @@
+PEP: 9999
+Title: imath --- module for number-theoretic functions
+Author: Sergey B Kirpichev
+Sponsor: Victor Stinner <vstinner@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 10-May-2025
+Python-Version: 3.15
+Post-History: 02-Jun-2019,
+              `09-May-2025 <https://discuss.python.org/t/91337>`__,
+
+
+Abstract
+========
+
+This PEP describes new module for number-theoretic functions, like
+:external+py3.14:func:`math.gcd` or :external+py3.14:func:`math.isqrt`.
+
+
+Motivation
+==========
+
+The :external+py3.14:mod:`math` documentaton says: "This module provides
+access to the mathematical functions defined by the C standard."  But as a
+state of art, over time the module was populated with functions that aren't
+related to the C standard or floating-point arithmetics.  Now it's much
+harder to describe module scope, content and interfaces (e.g. returned
+values).
+
+Apparently, the :external+py3.14:mod:`math` can't serve as a catch-all place
+for mathematical functions: we have also the :external+py3.14:mod:`cmath` and
+the :external+py3.14:mod:`statistics`.  Lets make same for integer-related
+functions.
+
+
+Specification
+=============
+
+The PEP proposes moving the following integer-related functions [1]_ to a new
+module, called ``imath``:
+
+* :external+py3.14:func:`~math.gcd`
+* :external+py3.14:func:`~math.lcm`
+* :external+py3.14:func:`~math.isqrt`
+* :external+py3.14:func:`~math.factorial`
+* :external+py3.14:func:`~math.comb`
```

Comment:
```suggestion
* :external+py3.14:func:`~math.perm`
* :external+py3.14:func:`~math.comb`
```

Reply:
Hmm, indeed.  Thanks.  This was blindly copied from #9.

Probably, it's better just add functions as they appear in the sphinx docs.